lib/active_admin/resource_controller/streaming.rb

```diff
@@ -24,7 +24,7 @@ def stream_resource(&block)
         headers["Last-Modified"] = Time.current.httpdate
 
         if ActiveAdmin.application.disable_streaming_in.include? Rails.env
-          self.response_body = block[""]
+          self.response_body = block[String.new]
```

Comment:
Curious since I'm not familiar with `frozen_string_literal` usage, what difference does `String.new` have over what this did previously?

Reply:
`String.new` or `+""` force the empty string they create to be mutable.

Reply:
While, in `frozen_string_literal: true` mode, `""` creates a frozen empty string.

Reply:
That makes sense now. Thanks David!